### DISCUSSION TITLE
Embedding Images

### DISCUSSION THREAD

**Discussion by kastyb** [reuseman/flashcards-obsidian#30] [Q&A] [answered]
The Plugin is awesome!
However I'm unable to view my embedded images in Anki. Can you explain "Be aware that an attachment folder must be specified in Obsidian." Somehow I'm getting something wrong in the process.

**Comment by reuseman** [comment, marked as answer]
Hey @kastyb, in the settings of Obsidian you need to specify a folder where all the media files go. This can be done in the following manner:
`Settings -> Files & Links -> Default location for new attachments -> In the folder specified below`
And you specify a folder, for example one named `media`

**Reply by kastyb** [reply]
Thanks for the fast reply!
